hydromt/models/model_api.py

```diff
@@ -640,7 +710,6 @@ def get_config(self, *args, fallback=None, abs_path=False):
         return value
 
     ## model parameter maps, geometries and spatial properties
-
     @property
     def staticmaps(self):
```

Comment:
In the future, maybe have another proerty to store as auxiliary or metadata (not necessary to have smae reoslution)